index.md

```diff
@@ -57,15 +57,16 @@ Looking for Homebrew or Windows installation? Checkout the [installation guide](
 If you are interested in more sophisticated use cases or want to develop modern Kubernetes-based architectures, 
 check out the [use cases and demos](/usecases/) or [the media section](/media/). 
 
-## Alternative to Telepresence 2
-Compared to [Telepresence 2](https://www.telepresence.io/docs/latest/reference/architecture/){:target="_blank"}, Gefyra uses a Wireguard-based
-VPN to connect with the Kubernetes cluster. Telepresence 2 provides a broad connectivity with the cluster ("your development
-machine becomes part of the cluster"), Gefyra instead establishes a very scoped connectivity based on a dedicated Docker-network on the
-developer machine. In addition, Gefyra supports a couple of important use-cases such as the sidecar pattern 
```

Comment:
I'd like to keep the part about sidecars in the comparison - as far as I know Telepresence does not support intercepting workloads which themselves use the sidecar pattern, does it?

Reply:
It does. The mutator webhook reinvocation policy can be controlled using our Helm chart. If it's set to `IfNeeded`, then our injector will be reinvoked after other sidecars have been injected.

Reply:
In this particular case, two containers communicate via localhost in two directions. The app calls the sidecar on localhost via a port-forward (that port-forward is unnecessary, this is what the `--to-pod <port>` flag is for), but then the sidecar creates a connection in the opposite direction, also using localhost. Telepresence cannot intercept a call made from a sidecar to localhost unless the sidecar too is intercepted. Simultaneous interception of multiple containers in one pod is supported though, so in theory, it should be possible to support use-cases like that.

Reply:
I added a commit that I think captures the difference better.

Side note: It sounds like Gefyra does what Telepresence 1 used to do, a "swap deployment". This is something [we've been discussing](https://github.com/telepresenceio/telepresence/issues/1608), because, as you point out, there are use-cases when it is desirable to simply replace the running container.